README.md

```
@@ -153,6 +153,79 @@ arrived yet; wait about a second before reading back a confirmed state. A
+# Watch presses without acting on them (by-path name comes from the UART
+# address, so it is board-specific; this is the Photonicat 2 one)
+sudo evtest /dev/input/by-path/platform-2afc0000.serial-event
+```
+
+The firmware debounces the button itself: on RA2E1260702000 a quick tap or a
+hold under about two seconds sends nothing at all, and a hold of about three
+seconds sends the request. A stray press in a bag therefore never reaches the
+host in the first place.
+
+> [!IMPORTANT]
+> The PMU applies [`force-poweroff-timeout`](#device-tree-properties) to a
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)